NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift

```diff
@@ -10,10 +10,10 @@ import UIKit
 
 class NVActivityIndicatorAnimationAudioEqualizer: NVActivityIndicatorAnimationDelegate {
     
-    func setUpAnimationInLayer(layer: CALayer, size: CGSize, color: UIColor) {
+    func setUpAnimationInLayer(_ layer: CALayer, size: CGSize, color: UIColor) {
```

Comment:
As API design guidelines, the method name should be `setUpAnimation(in layer: CALayer, size: CGSize, color: UIColor)`. What do you think?